Rocket.toml

```diff
@@ -6,8 +6,10 @@ key = "key.pem"
 address = "localhost"
 port = 8000
 log = "normal"
+secret_key = "u4shYQ/3Jm6+vh5QZsy4oX84rHDcS3/RDOgUEAL0p4M="
```

Comment:
Do not upload secret keys. They are secret.

Reply:
Oh sorry where do I put it then cause "Take the output of that command and then edit Rocket.toml so that the development and production sections have the following key:

secret_key = "[the generated key]", that's what you told me to do.

Reply:
Nvm I get what you were saying lol.  Don't upload it when I commit.